src/usbh_diskio.c

```diff
@@ -1,8 +1,8 @@
 #include "ff_gen_drv.h"
 #include "usbh_diskio.h"
-#include "timing.h"
 
 static const uint16_t USB_DEFAULT_BLOCK_SIZE = FF_MIN_SS;
+static DWORD scratch[FF_MAX_SS / 4];
```

Comment:
So in total this PR needs 2x 5KB + 1x 512B buffers, correct?

Reply:
Yes, plus the size of the prefetcher's stack. Either that or users of the FS need to align their buffers properly.